pkg/server/server.go

```diff
@@ -426,7 +428,12 @@ func (s *Server) handleHelmRenderManifests(user *auth.User, w http.ResponseWrite
 	params := r.URL.Query()
 	ns := params.Get("ns")
 	conf := helm.GetActionConfigurations(s.KubeAPIServerURL, ns, user.Token, &s.K8sClient.Transport)
-	resp, err := helm.RenderManifests(params.Get("name"), params.Get("url"), conf)
+	fValues := r.FormValue("values")
+	v := strings.Split(fValues, ",")
+	vals := &values.Options{
+		Values: v,
+	}
+	resp, err := helm.RenderManifests(params.Get("name"), params.Get("url"), vals, conf)
```

Comment:
Instead of packing params in URL, and as discussed the last week, it would be better to pack all of them in a json payload.

Reply:
Done